numpy/lib/function_base.py

```diff
@@ -2965,7 +2965,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
 
     >>> x = [-2.1, -1,  4.3]
     >>> y = [3,  1.1,  0.12]
-    >>> X = np.vstack((x,y))
+    >>> X = np.concatenate((x,y), axis=0)
```

Comment:
I don't understand the issue here. Do you mean if the input array has shape (x,)? That doesn't work as input to vstack. Can you give me some unit tests that will fail?

With (x,1) shapes it works as follows:

```
data = np.asarray(np.random.rand(2,1))
y = np.asarray(np.random.rand(4,1))

print np.vstack([data, y]).shape
print np.concatenate([data, y], axis=0).shape

print np.allclose(np.vstack([data, y]),np.concatenate([data, y], axis=0))
```
Output:
```
(6, 1)
(6, 1)
True
```